docs/caregiver-awards.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+---
+
+{% include header.md %}
+
+## Caregiver awards
+
+Small awards up to $500 _per family_ are available to _Bioconductor_ attendees
+who are either bringing young children to the meeting or who will be 
+incurring additional expenses for childcare or other caregiving responsibilities
+(such as elder care) during the meeting.
+
+### Allowable expenses
+
+- Child care expenses in Boston
+- Additional care expenses at home incurred because the primary caregiver was
+attending the meeting (such as overtime at childcare center, cost of a sitter or nurse ec)
+- Expenses incurred in bringing a caregiver and/or dependent
+
+**Receipts will be required to reimbursement**
+
+### Unallowable expenses
+
+- Regular care expenses, such as regular childcare
+- Travel or other expenses related to the attendee's participation in the meeting
+- Food
+- Tickets to local attractions
+
+### Selection of awards
+
+- If the number of requests for grants exceeds the funding, preference will be given to 
+applicants in earlier career stages.
+
+### Application misrepresentation
```

Comment:
I'm not sure that this is necessary; it leaves a kind of sour and distrustful note, and I'm not sure that we would act on any 'suspicious' claim anyway.

Reply:
Thanks, I just took that out!